tests/testthat/test-Image.R

```diff
@@ -18,3 +18,21 @@ test_that("Test Image getThumbnail",{
   expect_that(dim(thumb), equals(c(96,96,3)))
 })
 
+
+test_that("Test Image getPixelValues",{
+  img <- loadObject(server, "ImageData", imageID)
+
+  pixels <- getPixelValues(img, 1, 1, 1)
```

Comment:
have you checked value with invalid value i.e. 0, negative, value > sizeZ?

Reply:
I don't think so.
Maybe declare that an exception will be thrown if the specified values are out of range
so users know what to handle